KsApi/models/CommentRepliesEnvelope.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import ReactiveSwift
+
+public struct CommentRepliesEnvelope: Decodable {
```

Comment:
It's good that the `RepliesEnvelope` is separate from the `CommentsEnvelope` because it's a completely different screen and a brand new query even though the results are slightly different.